docs/flame-charts-with-cpu-profiles.mdx

```diff
@@ -1,12 +1,12 @@
-# Iciclecharts with CPU profiling data
+# Flame Charts with CPU profiling data
 
-Iciclechart is a visualization method that allows you to see the CPU usage of your application over time. In other words, it is similar to a flamegraph, but has an additonal time dimension. This means that you can see how the CPU usage of your application changes over time, and how it is affected by different events.
+The Flame Chart is a visualization method that allows you to see the CPU usage of your application over time. In other words, it is similar to a flame graph, but has an additonal time dimension. This means that you can see how the CPU usage of your application changes over time, and how it is affected by different events.
 
-Screenshot of Iciclechart with CPU profiling data:
-![Iciclechart with CPU profiling data](/img/vizualisation/iciclechart-example.png)
+Screenshot of Flame Chart with CPU profiling data:
+![Flame Chart with CPU profiling data](/img/vizualisation/iciclechart-example.png)
```

Comment:
I think we can rename the image as well, for the benefit of seo.

Reply:
done!